### PR TITLE
Update README for 4.15.0

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update README for 4.15.0
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,32 +8,36 @@
...
+Please note PrestaShop 1.6 and older are EOL.
 
 # Installation
 
 All versions can be found in the [releases list](https://github.com/PrestaShop/autoupgrade/releases).
 
 ## Create a module from source code
-* Clone (`git clone https://github.com/PrestaShop/autoupgrade.git`) or [download](https://github.com/PrestaShop/autoupgrade/archive/master.zip) the source code. You can also download a release **Source code** ([ex. v4.4.1](https://github.com/PrestaShop/autoupgrade/archive/v4.4.1.zip)). If you download a source code archive, you need extract the file and rename the extracted folder to **autoupgrade**
+
+If you download a ZIP archive that contain the source code or if you want to use the current state of the code, you need to build the module from the sources:
```
Comment (kpodemski) [inline README.md:29]:
```suggestion
If you download a ZIP archive that contains the source code or if you want to use the current state of the code, you need to build the module from the sources:
```

File: README.md [hunk 2]
```diff
@@ -85,7 +89,7 @@ You can report issues with this module in the main PrestaShop repository. [Click
 This module is released under the [Academic Free License 3.0][AFL-3.0]
 
 [report-issue]: https://github.com/PrestaShop/PrestaShop/issues/new/choose
-[prestashop]: https://www.prestashop.com/
-[contribution-guidelines]: https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/project-modules/
+[prestashop]: https://www.prestashop-project.org/
+[contribution-guidelines]: https://devdocs.prestashop-project.org/1.7/contribute/contribution-guidelines/project-modules/
 [AFL-3.0]: https://opensource.org/licenses/AFL-3.0
-[doc]: https://devdocs.prestashop.com/1.7/development/upgrade-module/
+[doc]: https://devdocs.prestashop-project.org/1.7/development/upgrade-module/
```
Comment (kpodemski) [inline README.md:95]:
```suggestion
[doc]: https://devdocs.prestashop-project.org/8/development/upgrade-module/
```

File: README.md [hunk 3]
```diff
@@ -85,7 +89,7 @@ You can report issues with this module in the main PrestaShop repository. [Click
 This module is released under the [Academic Free License 3.0][AFL-3.0]
 
 [report-issue]: https://github.com/PrestaShop/PrestaShop/issues/new/choose
-[prestashop]: https://www.prestashop.com/
-[contribution-guidelines]: https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/project-modules/
+[prestashop]: https://www.prestashop-project.org/
+[contribution-guidelines]: https://devdocs.prestashop-project.org/1.7/contribute/contribution-guidelines/project-modules/
```
Comment (kpodemski) [inline README.md:93]:
```suggestion
[contribution-guidelines]: https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/project-modules/
```

File: README.md [hunk 4]
```diff
@@ -8,32 +8,36 @@
...
+* Create a new ZIP archive from the of **autoupgrade** folder.
+* Now you can install it on your shop. For example you can upload it using the dropzone in Module Manager back-office page. 
 
 # Running an upgrade on PrestaShop
 
-Upgrading a shop can be done via:
+Upgrading a shop can be done using:
 
-* the configuration page of the module (access from your BO module page)
+* the configuration page of the module (browse the back-office page provided by the module)
```
Comment (kpodemski) [inline README.md:40]:
```suggestion
* the configuration page of the module (browse the back office page provided by the module)
```

File: README.md [hunk 5]
```diff
@@ -8,32 +8,36 @@
...
+
+If you download a ZIP archive that contain the source code or if you want to use the current state of the code, you need to build the module from the sources:
+
+* Clone (`git clone https://github.com/PrestaShop/autoupgrade.git`) or [download](https://github.com/PrestaShop/autoupgrade/archive/master.zip) the source code. You can also download a release **Source code** ([ex. v4.14.2](https://github.com/PrestaShop/autoupgrade/archive/v4.14.2.zip)). If you download a source code archive, you need to extract the file and rename the extracted folder to **autoupgrade**
 * Enter into folder **autoupgrade** and run the command `composer install`  ([composer](https://getcomposer.org/)).
-* Create a new zip file of **autoupgrade** folder
-* Now you can upload into your module pages
+* Create a new ZIP archive from the of **autoupgrade** folder.
+* Now you can install it on your shop. For example you can upload it using the dropzone in Module Manager back-office page.
```
Comment (kpodemski) [inline README.md:34]:
```suggestion
* Now you can install it in your shop. For example, you can upload it using the dropzone in Module Manager back office page. 
```

**Review by kpodemski — COMMENTED** [review]
A few small comments

**Comment by matks** [comment]
@kpodemski Suggestions added

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks 👌

**Pull request merged**